guardrails/validators.py

```diff
@@ -764,6 +766,62 @@ def validate(self, key: str, value: Any, schema: Union[Dict, List]) -> Dict:
         return schema
 
 
+class SQLDriver(ABC):
+    @abstractmethod
+    def validate_sql(self, query: str) -> List[str]:
+        ...
+
+
+class SimpleSqlDriver(SQLDriver):
+    def validate_sql(self, query: str) -> List[str]:
+        import sqlvalidator
+
+        sql_query = sqlvalidator.parse(query)
+        if not sql_query.is_valid():
+            return sql_query.errors
+        return sql_query.errors
+
+
+class SqlAlchemyDriver(SQLDriver):
+    def __init__(self, schema_file: Optional[str], conn: Optional[str]) -> None:
+        import sqlalchemy as db
+        from sqlalchemy import text
+
+        if schema_file is not None and conn is None:
+            raise ValidationError(
+                """schema_file should accompany a sql connection string for
+           guardrails to apply it to a database backend.
+           Use sqlite for ex: sqlite://"""
+            )
+
+        if conn is not None:
+            try:
+                self._engine = db.create_engine(conn)
+                self._conn = self._engine.connect()
+            except Exception as ex:
+                raise ValueError(ex)
+
+        if schema_file is not None:
```

Comment:
Can you add a check here to make sure that the db doesn't have existing data that would get overwritten by executing the schema?

Reply:
@ShreyaR I was assuming the schema_file was a mechanism for developers to "setup" a test instance of the database so they would know what they are doing. You think with an empty database check, users will be forced to point the validator to a test instance?